config/_ytt_lib/github.com/cloudfoundry/cf-k8s-networking/config/crd/networking.cloudfoundry.org_routes.yaml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
This whitespace was intentionally added. Kubebuilder adds that blank line to its generated CRDs for some reason (maybe not intentional on their end). See the example `CronJob` CRD:

https://github.com/kubernetes-sigs/kubebuilder/blob/d6e2b995831ce26228f2303e9d1b3a55acd3d225/docs/book/src/cronjob-tutorial/testdata/project/config/crd/bases/batch.tutorial.kubebuilder.io_cronjobs.yaml#L1

We had manually copy pasted it to our top level `config` directly previously I think which is why the blank line shows up in the diff here. This time the generated file was copied wholesale.